Dockerfile

```diff
@@ -35,4 +41,4 @@ RUN npm run build
 EXPOSE 3000
 
 # CMD is the default command when running the docker container.
-CMD [ "npm", "start" ]
\ No newline at end of file
+CMD [ "npm", "run", "start:newrelic" ]
```

Comment:
do we really need to update the task definitions/CFT? is this not enough to get this to work in qa/prod?

Reply:
I don't think the TD/CFT needs to be updated since it (I'm assuming) already has this command. I think it's helpful to have this here for consistency and so we better know what command runs in qa/prod.

Reply:
mmm I don't know if your assumption is correct but I think this line in the dockerfile should be enough. Think we should push this out and check the TD/CFT after if this doesn't fix the browser monitoring in qa - does that make sense to you?

Reply:
This won't work without the env var set in the task definition or the CF template. I think we can
1. push to QA to test and verify whether it works or not (most likely won't)
2. update the task definition to include the new env var for the app name. Test QA (I think this should work).
3. If it does, update the CF template with the env var so it persists on new server instance startups.

Reply:
ah I think I got confused by what actually needed to be updated in the cft/task defintion. I thought you were suggesting that we add the "npm run start:newrelic" to the td/cft.